diff -urN ushare-0.9.7.orig/scripts/Makefile.in ushare-0.9.7/scripts/Makefile.in
--- ushare-0.9.7.orig/scripts/Makefile.in	2006-03-12 14:06:02.000000000 +0100
+++ ushare-0.9.7/scripts/Makefile.in	2008-12-15 13:02:46.000000000 +0100
@@ -335,10 +335,10 @@
 
 
 install-data-local:
-	$(mkinstalldirs) $(DESTDIR)$(sysconfdir)/init.d
+	$(mkinstalldirs) $(DESTDIR)$(sysconfdir)/rc.d/init.d
 	@for s in $(init_scripts); do \
-		echo " $(INSTALL_SCRIPT) -m 755 $(srcdir)/$$s $(DESTDIR)$(sysconfdir)/init.d/$$s"; \
-		$(INSTALL_SCRIPT) -m 755 "$(srcdir)/$$s" "$(DESTDIR)$(sysconfdir)/init.d/$$s"; \
+		echo " $(INSTALL_SCRIPT) -m 755 $(srcdir)/$$s $(DESTDIR)$(sysconfdir)/rc.d/init.d/$$s"; \
+		$(INSTALL_SCRIPT) -m 755 "$(srcdir)/$$s" "$(DESTDIR)$(sysconfdir)/rc.d/init.d/$$s"; \
 	done
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
